octopusdeploy_framework/schemas/schema_maven_feed.go

```diff
@@ -0,0 +1,53 @@
+package schemas
+
+import (
+	"github.com/OctopusDeploy/terraform-provider-octopusdeploy/octopusdeploy_framework/util"
+	resourceSchema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/int64default"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+const mavenFeedDescription = "maven feed"
+
+func GetMavenFeedResourceSchema() map[string]resourceSchema.Attribute {
+	return map[string]resourceSchema.Attribute{
+		"download_attempts": resourceSchema.Int64Attribute{
+			Default:     int64default.StaticInt64(5),
+			Description: "The number of times a deployment should attempt to download a package from this feed before failing.",
+			Optional:    true,
+			Computed:    true,
+		},
+		"download_retry_backoff_seconds": resourceSchema.Int64Attribute{
+			Default:     int64default.StaticInt64(10),
+			Description: "The number of seconds to apply as a linear back off between download attempts.",
+			Optional:    true,
+			Computed:    true,
+		},
+		"feed_uri": resourceSchema.StringAttribute{
+			Required: true,
+		},
+		"id": util.GetIdResourceSchema(),
+		// Should this use the existing description? "A short, memorable, unique name for this feed. Example: ACME Builds.",
+		"name": util.GetNameResourceSchema(true),
+		"package_acquisition_location_options": resourceSchema.ListAttribute{
```

Comment:
These were set on the old schema, but are not respected. The API just returns ["Server", "ExecutionTarget"]. This can be dropped but I've kept it here for consistency with the old schema.